src/main/java/com/microsoft/jfr/FlightRecorderConnection.java

```diff
@@ -1,21 +1,8 @@
 package com.microsoft.jfr;
 
-import javax.management.InstanceNotFoundException;
-import javax.management.MBeanException;
-import javax.management.MBeanServerConnection;
-import javax.management.MalformedObjectNameException;
-import javax.management.ObjectInstance;
-import javax.management.ObjectName;
-import javax.management.ReflectionException;
-import javax.management.openmbean.CompositeData;
-import javax.management.openmbean.CompositeDataSupport;
-import javax.management.openmbean.CompositeType;
+import javax.management.*;
```

Comment:
Please revert to the fully qualified imports. You probably need to set your IDE to not use star imports or set the number of imports before it uses star to a really high number.

Reply:
Sorry, I use the IDE configuration for OTel projects